botorch/models/transforms/input.py

```diff
@@ -1098,7 +1098,9 @@ def __init__(
 
         Args:
             perturbation_set: An `n_p x d`-dim tensor denoting the perturbations
-                to be added to the inputs.
+                to be added to the inputs. Alternatively, this can be a callable that
```

Comment:
Can you update the type of the arg then to be `Union[Tensor, Callable[[Tensor], Tensor]`?